src/main/java/io/github/genomicdatainfrastructure/daam/exceptions/ApplicationSubmissionException.java

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+package io.github.genomicdatainfrastructure.daam.exceptions;
+
+import java.util.List;
+
+public class ApplicationSubmissionException extends RuntimeException {
+
+    private static final String MESSAGE = "Application %s could not be submitted";
+    private final List<String> errorMessages;
+
+    public ApplicationSubmissionException(Long applicationId, List<String> errorMessages) {
+        super(String.format(MESSAGE, applicationId) + (errorMessages == null || errorMessages
```

Comment:
important: can you please remove the errorMessages from the exception message?

Although it makes sense to concatenate the errors into it, this makes harder to the frontend to render the errors. It is better to send the error messages in the `ErrorResponse` body.